source/UberRides/RequestDeeplink.swift

```diff
@@ -32,14 +32,48 @@ import UberCore
  */
 @objc(UBSDKRequestDeeplink) public class RequestDeeplink: BaseDeeplink {
     static let sourceString = "deeplink"
-    
-    @objc public init(rideParameters: RideParameters = RideParametersBuilder().build()) {
-        rideParameters.source = rideParameters.source ?? RequestDeeplink.sourceString
-        
+
+    private let rideParameters: RideParameters
+    private let fallbackType: DeeplinkFallbackType
+
+    @objc public convenience init(rideParameters: RideParameters = RideParametersBuilder().build()) {
+        self.init(rideParameters: rideParameters, fallbackType: DeeplinkFallbackType.mobileWeb)
```

Comment:
The default should probably be app store, then mobile Web.

Reply:
+1

Reply:
I feel like we should encourage more muber usage, but it's up to the team!